pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -2549,4 +2549,7 @@ type KernelModuleConfig struct {
 	// description: |
 	//   Module name.
 	ModuleName string `yaml:"name"`
+	// description: |
+	//   Module parameters.
```

Comment:
we might want to add a note here that changing module parameters requires a reboot